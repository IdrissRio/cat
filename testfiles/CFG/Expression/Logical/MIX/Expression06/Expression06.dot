digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[22]"; "Opt[25]"; "List[26]"; "List[27]"; "InstanceInitialization[6]"; "InstanceInitialization[14]"}
{ rank = same; "List[23]"; "MethodDecl[28]"; "Entry[7]"; "Exit[8]"; "Entry[15]"; "Exit[16]"}
{ rank = same; "Modifier[24]"; "Entry[29]"; "Modifiers[30]"; "PrimitiveTypeAccess[32]"; "List[33]"; "List[42]"; "Opt[43]"; "Exit[44]"}
{ rank = same; "List[31]"; "ParameterDeclaration[34]"; "ParameterDeclaration[38]"; "Block[44]"}
{ rank = same; "Modifiers[35]"; "PrimitiveTypeAccess[37]"; "Modifiers[39]"; "PrimitiveTypeAccess[41]"; "List[45]"}
{ rank = same; "List[36]"; "List[40]"; "VarDeclStmt[46]"}
{ rank = same; "Modifiers[47]"; "ArrayTypeAccess[49]"; "List[52]"}
{ rank = same; "List[48]"; "ArrayTypeAccess[50]"; "VariableDeclarator[53]"}
{ rank = same; "PrimitiveTypeAccess[51]"; "List[54]"; "Opt[55]"}
{ rank = same; "ArrayInit[56]"}
{ rank = same; "List[57]"}
{ rank = same; "List[59]"; "List[75]"}
{ rank = same; "ArrayInit[58]"; "ArrayInit[74]"}
{ rank = same; "GTExpr[61]"; "LTExpr[64]"; "GTExpr[68]"; "LTExpr[71]"; "GTExpr[77]"; "LTExpr[80]"}
{ rank = same; "AndLogicalExpr[60]"; "OrLogicalExpr[67]"; "AndLogicalExpr[76]"}
{ rank = same; "VarAccess[62]"; "IntegerLiteral[63]"; "VarAccess[65]"; "IntegerLiteral[66]"; "VarAccess[69]"; "IntegerLiteral[70]"; "VarAccess[72]"; "IntegerLiteral[73]"; "VarAccess[78]"; "IntegerLiteral[79]"; "VarAccess[81]"; "IntegerLiteral[82]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Expression06 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[22]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[23]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[24]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[25]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[27]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[28]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[29]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[30]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[31]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[32]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[34]" [label="ParameterDeclaration\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[35]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[37]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[38]" [label="ParameterDeclaration\n < c >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[39]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[41]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[42]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[43]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[44]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[45]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[46]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[47]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[48]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayTypeAccess[49]" [label="ArrayTypeAccess\n < boolean[] >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayTypeAccess[50]" [label="ArrayTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[51]" [label="PrimitiveTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[52]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[53]" [label="VariableDeclarator\n < array >", fillcolor=white   style=filled  ]
"List[54]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[55]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayInit[56]" [label="ArrayInit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[57]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayInit[58]" [label="ArrayInit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[59]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[60]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[61]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[62]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[63]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[64]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[65]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[66]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"OrLogicalExpr[67]" [label="OrLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[68]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[69]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[70]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[71]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[72]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[73]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"ArrayInit[74]" [label="ArrayInit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[75]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[76]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[77]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[78]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[79]" [label="IntegerLiteral\n < 5 >", fillcolor=white   style=filled  ]
"LTExpr[80]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[81]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[82]" [label="IntegerLiteral\n < 6 >", fillcolor=white   style=filled  ]
"Exit[44]" [label="Exit\n ", fillcolor=white   style=filled  ]
"InstanceInitialization[6]" [label="InstanceInitialization\n < Instance >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[7]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Exit[8]" [label="Exit\n ", fillcolor=white   style=filled  ]
"InstanceInitialization[14]" [label="InstanceInitialization\n < Static >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[15]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Exit[16]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[23]" -> "Modifier[24]" [style=dashed, color=gray]
"Modifiers[22]" -> "List[23]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[22]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[25]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[26]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Entry[29]" [style=dashed, color=gray]
"Modifiers[30]" -> "List[31]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Modifiers[30]" [style=dashed, color=gray]
"MethodDecl[28]" -> "PrimitiveTypeAccess[32]" [style=dashed, color=gray]
"Modifiers[35]" -> "List[36]" [style=dashed, color=gray]
"ParameterDeclaration[34]" -> "Modifiers[35]" [style=dashed, color=gray]
"ParameterDeclaration[34]" -> "PrimitiveTypeAccess[37]" [style=dashed, color=gray]
"List[33]" -> "ParameterDeclaration[34]" [style=dashed, color=gray]
"Modifiers[39]" -> "List[40]" [style=dashed, color=gray]
"ParameterDeclaration[38]" -> "Modifiers[39]" [style=dashed, color=gray]
"ParameterDeclaration[38]" -> "PrimitiveTypeAccess[41]" [style=dashed, color=gray]
"List[33]" -> "ParameterDeclaration[38]" [style=dashed, color=gray]
"MethodDecl[28]" -> "List[33]" [style=dashed, color=gray]
"MethodDecl[28]" -> "List[42]" [style=dashed, color=gray]
"Modifiers[47]" -> "List[48]" [style=dashed, color=gray]
"VarDeclStmt[46]" -> "Modifiers[47]" [style=dashed, color=gray]
"ArrayTypeAccess[50]" -> "PrimitiveTypeAccess[51]" [style=dashed, color=gray]
"ArrayTypeAccess[49]" -> "ArrayTypeAccess[50]" [style=dashed, color=gray]
"VarDeclStmt[46]" -> "ArrayTypeAccess[49]" [style=dashed, color=gray]
"VariableDeclarator[53]" -> "List[54]" [style=dashed, color=gray]
"GTExpr[61]" -> "VarAccess[62]" [style=dashed, color=gray]
"GTExpr[61]" -> "IntegerLiteral[63]" [style=dashed, color=gray]
"AndLogicalExpr[60]" -> "GTExpr[61]" [style=dashed, color=gray]
"LTExpr[64]" -> "VarAccess[65]" [style=dashed, color=gray]
"LTExpr[64]" -> "IntegerLiteral[66]" [style=dashed, color=gray]
"AndLogicalExpr[60]" -> "LTExpr[64]" [style=dashed, color=gray]
"List[59]" -> "AndLogicalExpr[60]" [style=dashed, color=gray]
"GTExpr[68]" -> "VarAccess[69]" [style=dashed, color=gray]
"GTExpr[68]" -> "IntegerLiteral[70]" [style=dashed, color=gray]
"OrLogicalExpr[67]" -> "GTExpr[68]" [style=dashed, color=gray]
"LTExpr[71]" -> "VarAccess[72]" [style=dashed, color=gray]
"LTExpr[71]" -> "IntegerLiteral[73]" [style=dashed, color=gray]
"OrLogicalExpr[67]" -> "LTExpr[71]" [style=dashed, color=gray]
"List[59]" -> "OrLogicalExpr[67]" [style=dashed, color=gray]
"ArrayInit[58]" -> "List[59]" [style=dashed, color=gray]
"List[57]" -> "ArrayInit[58]" [style=dashed, color=gray]
"GTExpr[77]" -> "VarAccess[78]" [style=dashed, color=gray]
"GTExpr[77]" -> "IntegerLiteral[79]" [style=dashed, color=gray]
"AndLogicalExpr[76]" -> "GTExpr[77]" [style=dashed, color=gray]
"LTExpr[80]" -> "VarAccess[81]" [style=dashed, color=gray]
"LTExpr[80]" -> "IntegerLiteral[82]" [style=dashed, color=gray]
"AndLogicalExpr[76]" -> "LTExpr[80]" [style=dashed, color=gray]
"List[75]" -> "AndLogicalExpr[76]" [style=dashed, color=gray]
"ArrayInit[74]" -> "List[75]" [style=dashed, color=gray]
"List[57]" -> "ArrayInit[74]" [style=dashed, color=gray]
"ArrayInit[56]" -> "List[57]" [style=dashed, color=gray]
"Opt[55]" -> "ArrayInit[56]" [style=dashed, color=gray]
"VariableDeclarator[53]" -> "Opt[55]" [style=dashed, color=gray]
"List[52]" -> "VariableDeclarator[53]" [style=dashed, color=gray]
"VarDeclStmt[46]" -> "List[52]" [style=dashed, color=gray]
"List[45]" -> "VarDeclStmt[46]" [style=dashed, color=gray]
"Block[44]" -> "List[45]" [style=dashed, color=gray]
"Opt[43]" -> "Block[44]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Opt[43]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Exit[44]" [style=dashed, color=gray]
"List[27]" -> "MethodDecl[28]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[27]" [style=dashed, color=gray]
"InstanceInitialization[6]" -> "Entry[7]" [style=dashed, color=gray]
"InstanceInitialization[6]" -> "Exit[8]" [style=dashed, color=gray]
"InstanceInitialization[14]" -> "Entry[15]" [style=dashed, color=gray]
"InstanceInitialization[14]" -> "Exit[16]" [style=dashed, color=gray]
"ClassDecl[5]" -> "InstanceInitialization[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "InstanceInitialization[14]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[16]" -> "Entry[15]" [color=red, constraint=false]
"Entry[15]" -> "Exit[16]" [color=blue, constraint=false, xlabel=" "] 
"Exit[8]" -> "Entry[7]" [color=red, constraint=false]
"Entry[7]" -> "Exit[8]" [color=blue, constraint=false, xlabel=" "] 
"Exit[44]" -> "VariableDeclarator[53]" [color=red, constraint=false]
"VariableDeclarator[53]" -> "Exit[44]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[53]" -> "GTExpr[77]" [color=red, constraint=false]
"VariableDeclarator[53]" -> "LTExpr[80]" [color=red, constraint=false]
"LTExpr[80]" -> "VariableDeclarator[53]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[80]" -> "IntegerLiteral[82]" [color=red, constraint=false]
"IntegerLiteral[82]" -> "LTExpr[80]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[82]" -> "VarAccess[81]" [color=red, constraint=false]
"VarAccess[81]" -> "IntegerLiteral[82]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[81]" -> "GTExpr[77]" [color=red, constraint=false]
"GTExpr[77]" -> "VarAccess[81]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[77]" -> "VariableDeclarator[53]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[77]" -> "IntegerLiteral[79]" [color=red, constraint=false]
"IntegerLiteral[79]" -> "GTExpr[77]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[79]" -> "VarAccess[78]" [color=red, constraint=false]
"VarAccess[78]" -> "IntegerLiteral[79]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[78]" -> "GTExpr[68]" [color=red, constraint=false]
"VarAccess[78]" -> "LTExpr[71]" [color=red, constraint=false]
"LTExpr[71]" -> "VarAccess[78]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[71]" -> "IntegerLiteral[73]" [color=red, constraint=false]
"IntegerLiteral[73]" -> "LTExpr[71]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[73]" -> "VarAccess[72]" [color=red, constraint=false]
"VarAccess[72]" -> "IntegerLiteral[73]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[72]" -> "GTExpr[68]" [color=red, constraint=false]
"GTExpr[68]" -> "VarAccess[78]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[68]" -> "VarAccess[72]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[68]" -> "IntegerLiteral[70]" [color=red, constraint=false]
"IntegerLiteral[70]" -> "GTExpr[68]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[70]" -> "VarAccess[69]" [color=red, constraint=false]
"VarAccess[69]" -> "IntegerLiteral[70]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[69]" -> "GTExpr[61]" [color=red, constraint=false]
"VarAccess[69]" -> "LTExpr[64]" [color=red, constraint=false]
"LTExpr[64]" -> "VarAccess[69]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[64]" -> "IntegerLiteral[66]" [color=red, constraint=false]
"IntegerLiteral[66]" -> "LTExpr[64]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[66]" -> "VarAccess[65]" [color=red, constraint=false]
"VarAccess[65]" -> "IntegerLiteral[66]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[65]" -> "GTExpr[61]" [color=red, constraint=false]
"GTExpr[61]" -> "VarAccess[65]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[61]" -> "VarAccess[69]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[61]" -> "IntegerLiteral[63]" [color=red, constraint=false]
"IntegerLiteral[63]" -> "GTExpr[61]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[63]" -> "VarAccess[62]" [color=red, constraint=false]
"VarAccess[62]" -> "IntegerLiteral[63]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[62]" -> "Entry[29]" [color=red, constraint=false]
"Entry[29]" -> "VarAccess[62]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"InstanceInitialization[14]" -> "InstanceInitialization[6]" -> "Modifiers[22]" -> "Opt[25]" -> "List[26]" -> "List[27]" [style=invis]
"Entry[29]" -> "Modifiers[30]" -> "PrimitiveTypeAccess[32]" -> "List[33]" -> "List[42]" -> "Opt[43]" -> "Exit[44]" [style=invis]
"ParameterDeclaration[34]" -> "ParameterDeclaration[38]" [style=invis]
"Modifiers[35]" -> "PrimitiveTypeAccess[37]" [style=invis]
"Modifiers[39]" -> "PrimitiveTypeAccess[41]" [style=invis]
"Modifiers[47]" -> "ArrayTypeAccess[49]" -> "List[52]" [style=invis]
"List[54]" -> "Opt[55]" [style=invis]
"ArrayInit[58]" -> "ArrayInit[74]" [style=invis]
"AndLogicalExpr[60]" -> "OrLogicalExpr[67]" [style=invis]
"GTExpr[61]" -> "LTExpr[64]" [style=invis]
"VarAccess[62]" -> "IntegerLiteral[63]" [style=invis]
"VarAccess[65]" -> "IntegerLiteral[66]" [style=invis]
"GTExpr[68]" -> "LTExpr[71]" [style=invis]
"VarAccess[69]" -> "IntegerLiteral[70]" [style=invis]
"VarAccess[72]" -> "IntegerLiteral[73]" [style=invis]
"GTExpr[77]" -> "LTExpr[80]" [style=invis]
"VarAccess[78]" -> "IntegerLiteral[79]" [style=invis]
"VarAccess[81]" -> "IntegerLiteral[82]" [style=invis]
"Entry[7]" -> "Exit[8]" [style=invis]
"Entry[15]" -> "Exit[16]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
74", fillcolor=1]
Edges [label="Total Nr. Edges
123", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
23", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
25", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}