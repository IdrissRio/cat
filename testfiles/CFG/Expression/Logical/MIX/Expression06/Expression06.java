public class Expression06 {
  void m(int a, int c) {
    boolean[][] array = {{a > 0 && c<0, a> 0 || c < 0}, {a > 5 && c < 6}};
  }
}