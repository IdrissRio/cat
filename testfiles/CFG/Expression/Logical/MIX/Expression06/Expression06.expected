NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [VarAccess[62]]
NodeName: VarAccess[62] Successors: [IntegerLiteral[63]]
NodeName: IntegerLiteral[63] Successors: [GTExpr[61]]
NodeName: GTExpr[61] Successors: [VarAccess[65], VarAccess[69]]
NodeName: VarAccess[65] Successors: [IntegerLiteral[66]]
NodeName: IntegerLiteral[66] Successors: [LTExpr[64]]
NodeName: LTExpr[64] Successors: [VarAccess[69]]
NodeName: VarAccess[69] Successors: [IntegerLiteral[70]]
NodeName: IntegerLiteral[70] Successors: [GTExpr[68]]
NodeName: GTExpr[68] Successors: [VarAccess[78], VarAccess[72]]
NodeName: VarAccess[78] Successors: [IntegerLiteral[79]]
NodeName: IntegerLiteral[79] Successors: [GTExpr[77]]
NodeName: GTExpr[77] Successors: [VarAccess[81], VariableDeclarator[53]]
NodeName: VarAccess[81] Successors: [IntegerLiteral[82]]
NodeName: IntegerLiteral[82] Successors: [LTExpr[80]]
NodeName: LTExpr[80] Successors: [VariableDeclarator[53]]
NodeName: VariableDeclarator[53] Successors: [Exit[44]]
NodeName: Exit[44] Successors: []
NodeName: VarAccess[72] Successors: [IntegerLiteral[73]]
NodeName: IntegerLiteral[73] Successors: [LTExpr[71]]
NodeName: LTExpr[71] Successors: [VarAccess[78]]

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[44] Predecessor: [VariableDeclarator[53]]
NodeName: VariableDeclarator[53] Predecessor: [GTExpr[77], LTExpr[80]]
NodeName: GTExpr[77] Predecessor: [IntegerLiteral[79]]
NodeName: IntegerLiteral[79] Predecessor: [VarAccess[78]]
NodeName: VarAccess[78] Predecessor: [GTExpr[68], LTExpr[71]]
NodeName: GTExpr[68] Predecessor: [IntegerLiteral[70]]
NodeName: IntegerLiteral[70] Predecessor: [VarAccess[69]]
NodeName: VarAccess[69] Predecessor: [GTExpr[61], LTExpr[64]]
NodeName: GTExpr[61] Predecessor: [IntegerLiteral[63]]
NodeName: IntegerLiteral[63] Predecessor: [VarAccess[62]]
NodeName: VarAccess[62] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: LTExpr[64] Predecessor: [IntegerLiteral[66]]
NodeName: IntegerLiteral[66] Predecessor: [VarAccess[65]]
NodeName: VarAccess[65] Predecessor: [GTExpr[61]]
NodeName: LTExpr[71] Predecessor: [IntegerLiteral[73]]
NodeName: IntegerLiteral[73] Predecessor: [VarAccess[72]]
NodeName: VarAccess[72] Predecessor: [GTExpr[68]]
NodeName: LTExpr[80] Predecessor: [IntegerLiteral[82]]
NodeName: IntegerLiteral[82] Predecessor: [VarAccess[81]]
NodeName: VarAccess[81] Predecessor: [GTExpr[77]]
